packages/docs/operations-manual/validator-explorer.md

```diff
@@ -1,14 +1,14 @@
 
-A critical element of this explorer is the Validator Group name, which can help voters recognize organizations or active community members. This name is fetched from the Validator Group’s associated `metadata`, which you can read more about [here](https://github.com/celo-org/celo-monorepo/blob/master/packages/docs/celo-codebase/protocol/identity/metadata.md). In order to prevent name impersonation, a group can register a domain claim within their metadata. This is done by adding a [TXT record](https://en.wikipedia.org/wiki/TXT_record) to their domain which includes a signature of their domain claim signed by their associated account. This claim is then verified by the validator explorer. Individual users may also verify a claim using `celocli account:get-metdata` or by manually checking for this `TXT Record` on a group’s purported identity domain. 
+A critical element of this explorer is the Validator Group name, which can help voters recognize organizations or active community members. This name is fetched from the `account` information registered on-chain for your validator and validator group. In order to combat name impersonation, a group can register a domain claim within their metadata, which you can read more about [here](https://github.com/celo-org/celo-monorepo/blob/master/packages/docs/celo-codebase/protocol/identity/metadata.md). This verification is done by adding a [TXT record](https://en.wikipedia.org/wiki/TXT_record) to their domain which includes a signature of their domain claim signed by their associated account. This claim is then verified by the validator explorer. Individual users may also verify a claim using `celocli account:get-metdata` or by manually checking for this `TXT Record` on a group’s purported identity domain. 
 
-For example, if a group was run by the owners of `example.com`, they may want to register their Validator Group with the name `Example.com`. Additionally, they may want to add a DNS claim so no one can impersonate their valuable brand name. They can do this by adding a DNS claim to their metadata, claiming the URL `example.com`, while simultaneously adding a `TXT Record` to `example.com` that includes this claim signed by their group address. Let’s go through this example in detail, using a `ReleaseGold` contract as our validator group.
+For example, if a group was run by the owners of `example.com`, they may want to register their Validator Group with the name `Example.com`. The name does not need to be the same as the name of your domain, but for simplicity we do so here. To give credence to this name claim, they may want to add a DNS claim. They can do this by adding a DNS claim to their metadata, claiming the URL `example.com`, while simultaneously adding a `TXT Record` to `example.com` that includes this claim signed by their group address. Let’s go through this example in detail, using a `ReleaseGold` contract as our validator group.
```

Comment:
I would suggest the name of the group to be `Example` not `Example.com`. Also while this is technically a name claim, in this page we use `claim` in the metadata claim sense, not so much in the literal sense of the word, so I would probably just change `To give credence to this name claim` ====> `To give credence to this name`